pkg/disk/utils.go

```diff
@@ -1102,6 +1103,19 @@ func getDiskVolumeOptions(req *csi.CreateVolumeRequest) (*diskVolumeArgs, error)
 		diskVolArgs.ResourceGroupID = ""
 	}
 
+	// volumeSizeAutoAvailable
+	value, ok = volOptions["volumeSizeAutoAvailable"]
+	if !ok {
+		diskVolArgs.VolumeSizeAutoAvailable = false
```

Comment:
Also, here should be true as default.

Reply:
@fredkan The plan was to not enable this by default and for this reason we set this to false by default.